downloads/swan-lake-release-notes/swan-lake-bug-fixes-template/RELEASE_NOTE.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: ballerina-left-nav-release-notes
+title: Swan Lake Update 8 (<RELEASE_VERSION>) 
+permalink: /downloads/swan-lake-release-notes/<RELEASE_VERSION>/
+active: <RELEASE_VERSION>
+redirect_from: 
+    - /downloads/swan-lake-release-notes/<RELEASE_VERSION>
+    - /downloads/swan-lake-release-notes/<RELEASE_VERSION>/
+    - /downloads/swan-lake-release-notes/<RELEASE_VERSION>-swan-lake/
+    - /downloads/swan-lake-release-notes/<RELEASE_VERSION>-swan-lake
+---
+
+## Overview of Ballerina Swan Lake Update 8 (<RELEASE_VERSION>)
+
+<em>Swan Lake Update 8 (<RELEASE_VERSION>) is the <RELEASE_SEQUENCE_NUMBER> patch release of Ballerina 2201.8.0 (Swan Lake Update 8) and it includes a new set of bug fixes to the <ADD_THE_LIST_OF_COMPONENTS>.</em>
+
+## Update Ballerina
+
+Update your current Ballerina installation directly to <RELEASE_VERSION> by using the [Ballerina Update Tool](/learn/update-tool/) as follows.
+
+1. Run `bal update` to get the latest version of the Update Tool.
+2. Run `bal dist pull <RELEASE_VERSION>` to update to this latest distribution.
```

Comment:
Lets remove `bal update` and include `bal dist update` only

Reply:
Updated.